### ISSUE TITLE
No modules showing up

### ISSUE THREAD

**Issue by KCross84** [freescout-helpdesk/freescout#368]
Hello, I've just purchased about 6 of the modules available on your website and unfortunately when I go into Manage>Modules, the only thing that shows up is an invitation to register against FeedBurner to be notified when new modules are available. There's no modules to select, nor is there any place for a license key (I have not received any of those). I love this product as it definitely fills a gap in my company's workflow, but could someone please help me get these modules working? Thanks!

**Comment by freescout-helpdesk** [comment]
Make sure your server is able to access https://freescout.net/wp-json/

**Comment by KCross84** [comment]
Unfortunately, my server can't. I'm using shared hosting. I'm able to access it from my local machine, however.
![Screenshot_20191115-051742958_1](https://user-images.githubusercontent.com/47312078/68939936-7ab61280-0767-11ea-8741-9a996d7c723f.jpg)

**Comment by freescout-helpdesk** [comment]
Try to change temporarily in `/config/app.php` line 161 to:
```
'freescout_api' => 'https://cdn.freescout.net/wp-json/',
```

**Comment by KCross84** [comment]
Unfortunately, that has the same effect. Blank screen, invitation to sign up for notifications, etc. Here's the output from my ping. It looks like my server can access cdn.freescout.net but cannot access the SSL version. I don't know what causes that.

PING cdn.freescout.net (104.27.148.193) 56(84) bytes of data.
64 bytes from 104.27.148.193: icmp_seq=1 ttl=56 time=11.9 ms
--- cdn.freescout.net ping statistics ---
1 packets transmitted, 1 received, 0% packet loss, time 797ms
rtt min/avg/max/mdev = 11.914/11.914/11.914/0.000 ms

PING https://cdn.freescout.net
ping: unknown host https://cdn.freescout.net

**Comment by freescout-helpdesk** [comment]
Have you cleared the cache in System --> Tools?

Ping with protocol will not work. Try `curl https://cdn.freescout.net/wp-json/`

**Comment by KCross84** [comment]
I cleared the cache and it didn't work at first, but I just tried it and the modules show up, however, when I enter my key, I get the following error after a few minutes: cURL error 7: Failed to connect to freescout.net port 443: Connection timed out. 

There is a link to download the module manually, however, it takes me to a 404 page on your WP site.

**Comment by KCross84** [comment]
I'm still not sure what I am doing incorrectly. I've followed the instructions to the letter and I still cannot get the modules to load, nor can I download them manually.

**Comment by freescout-helpdesk** [comment]
> cURL error 7: Failed to connect to freescout.net port 443: Connection timed out.

It means this line in /config/app.php has not been applied or still cached:
```
'freescout_api' => 'https://cdn.freescout.net/wp-json/',
```
as cURL is addressing the old address.

Also try to contact your hosting to figure out why curl can not connect to https://freescout.net

**Comment by KCross84** [comment]
I changed it back and cleared the cache, and the modules wouldn't appear any longer. I changed it to the cdn address, cleared the cache, and the modules came back. I'm only running into problems when I try pasting my license key into the field and clicking "Install Module"--it always times out. I am able to cURL to the address with no issues, so I'm at a loss. I will contact my hosting provider and ask if they can help with getting to the regular address. 

Regarding the text "Please download module manually and extract", is there any way to do that? I click the link, but it takes me to a 404 page on the Wordpress installation.

Thanks for all your help. I'm looking forward to getting the plugins up and running.

Kirk

**Comment by KCross84** [comment]
Also, I'm not sure if this has anything at all to do with my issue, but on the System Status screen, there's this error for queue:work: Last run: ? Last successful run: ? Try to clear cache to force command to start. -- I've put the crontab together and the freescout:fetch-emails is running fine, but I don't know about queue:work. What do I need to do to make that run?

**Comment by freescout-helpdesk** [comment]
Just send a message to support@freescout.net and we will send you modules.
```
queue:work: Last run: ? Last successful run: ?
```
It means cron job is not working properly, check logs.

**Comment by KCross84** [comment]
Okay, got the modules and licensing actually worked, but now when I try to activate them, my system is complaining about symlinks being disabled. After researching this, it seems to be a mixture of my hosting provider locking down symlink and the Voyager framework not liking it. We've gotten this far—I didn't even think my modules would sync up, but they did. What would you recommend? Should we keep working at this? I mean, I can't forsee much more going awry.

**Comment by KCross84** [comment]
Okay, got the modules and licensing actually worked, but now when I try to activate them, my system is complaining about symlinks being disabled. After researching this, it seems to be a mixture of my hosting provider locking down symlink and the Voyager framework not liking it.

![Screenshot_20191116-025900889_1](https://user-images.githubusercontent.com/47312078/68990860-d8a53180-081d-11ea-8acd-ffe3595f7b67.jpg)

We've gotten this far—I didn't even think my modules would sync up, but they did. What would you recommend? Should we keep working at this? I mean, I can't forsee much more going awry.

**Comment by freescout-helpdesk** [comment]
Are you using the latest version of the FreeScout?

**Comment by KCross84** [comment]
Yes. I installed it using my webhost's Softaculous installer. 
It's version 1.3.10.

**Comment by freescout-helpdesk** [comment]
Try version 1.3.11

**Comment by KCross84** [comment]
My webhost has now disabled FreeScout in Softaculous. This seems to have happened within the last two days for some reason. I think I'm going to check out other options. Would you be so kind as to tell me how I may obtain a refund for the modules I purchased last week? I'm very sorry this didn't work out, and I sincerely appreciate your kind assistance. I guess my webhost figured out that they weren't as compatible as they thought.
![image](https://user-images.githubusercontent.com/47312078/69083660-2e3d3200-0a08-11ea-83af-71c595d96a1d.png)

**Comment by KCross84** [comment]
My webhost has actually removed this from Softaculous as an option for install. I tried Softaculous remote, and while the one-click install goes to a page with 1.3.11 available, when you go to install it, it's still 1.3.10. I don't have as much access as I'd need to install it from source, so unfortunately I'm going to need to seek out other options. I do thank you for your help, however.

Who do I need to email in order to obtain a refund for the modules I purchased?

Thanks again
Kirk
